README.md

```diff
@@ -52,6 +51,100 @@ as they will make continuous verification of your plugin's functionality much ea
 In order to use these tests, you will need to address the "FIXME" messages left in places where
 plugin-writer intervention is required.
 
+# Generate a template_config.yaml for an existing plugin
+
+The ``plugin-template`` script can update a plugin with the latest Travis CI configuration. Plugin
+authors can select which parts of the CI pipeline are included by providing a template_config.yaml
+in the root of the plugin repository. Run the following command to generate a template_config.yaml
+with all the defaults.
+
+    ``$ ./plugin-template --generate-config plugin_name plugin_app_label``
```

Comment:
I think it is not completely clear from the description that this command is not used in the root of the plugin but in the root of plugin_template and expects the plugin at `../<plugin_name>` (i.e. it generates `../<plugin_name>/template_config.yml`.